### DISCUSSION TITLE
Counter malicious attacks

### DISCUSSION THREAD

**Discussion by ProjectBay** [plausible/analytics#3000] [Anything Else]
Hi!

Im currently trying to integrate plausible.io into my webapp.
In their [docs](https://plausible.io/docs/events-api) for the event api it quotes

> The Plausible Events API can be used to record pageviews and custom events. This is useful when tracking Android or iOS mobile apps, or for server side tracking.

to trigger a plausible event.
If I mock that, for instance via postman, i can set the correct headers and body with a custom goal to increase an events count. This works fine.

BUT, when I integrate the frontend tracking into my web application, my plausible site name is exposed in the network tab. Therefore anyone can too now trigger any events count and disrupt my tracking numbers.
Is there a way to counter this?

I want to track the pageviews with a frontend-integration and business-logic events on my server.

**Comment by metmarkosaric** [comment]
thanks for your interest @ProjectBay! yes, this is correct and Plausible is not unique in that sense. with any web analytics tool you can send traffic very easily if you want to do so. but despite it being possible to do, it doesn't really happen that much so we haven't introduced any type of way to counter it at this stage
